libraries/incubator/ORM/Entity/EntityBuilder.php

```diff
@@ -385,11 +382,12 @@ public function handleStorage(Element $storage)
 					{
 						case 'csv':
 							$handler = '\Joomla\ORM\Storage\CsvProvider';
-							$param   = $parts[1];
+							break;
+						case 'orm':
+							$handler = '\Joomla\ORM\Storage\Doctrine\DoctrineProvider';
```

Comment:
So you want to prevent it to be able to define additional parameters? Can't understand why? Now it is more flexible.